### PR TITLE
It's catpull request 

### PR DESCRIPTION
**Pull request by laneleka**

### CODE REVIEW

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Well done! But your code affects other people's code. Possibly due to IDE settings.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Well done! But you don't have to affect others peoples code. I mean line 1607, 1612, 1613.

File: README.md [hunk 1]
```diff
@@ -1603,12 +1603,26 @@ Cheers, don't worry and be happy!
...
+
+## Hi everyone!
+
+My name is [Helen](https://github.com/laneleka) and this is my cat Cat.
+
+Glad to join and good luck, kottans!!!
+
+![CatCat](assets/images/cat-Cat.jpg)
+
+---
```
Comment (miroslav-kolomiets) [inline README.md:1628]: You only need to keep these changes.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (laneleka) [reply README.md:1626]: I fixed, thanks.

**Review by miroslav-kolomiets — APPROVED** [review]
Well done!

**Pull request merged**